app/controllers/api/v1/plans/applied_taxes_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Plans
+      class AppliedTaxesController < Api::BaseController
+        def create
+          return not_found_error(resource: 'plan') unless plan
```

Comment:
Since we already handle the "not_found" case in the create/destroy service, I'm wondering if we could get rid of these ones in the controller?

The `render_error_response` should take care of it automatically. What do you think?

Reply:
Good idea 👍